pyroomacoustics/bss/tests/test_bss.py

```diff
@@ -90,24 +93,32 @@ def freq_bss(algo='auxiva', L=256):
         S = np.sort(np.argpartition(average, -k)[-k:])
         # Run SparseAuxIva
         Y = pra.bss.sparseauxiva(X, S, n_iter=30, proj_back=True)
-        max_mse = 1e-4
+        max_mse = 1e-1
+    elif algo == 'overiva':
+        Y = pra.bss.auxiva(X, n_src=1, n_iter=30, proj_back=True)
+        max_mse = 0.5
+    print(Y.shape)
```

Comment:
Is this for debugging purposes or to generate a test log?
For the test log, it could be interesting to get some algorithm speed results.

Reply:
leftover debug...